.github/workflows/verify_semantic_pr.yml

```
@@ -30,10 +35,13 @@ permissions:
 jobs:
   semantic_pr:
     name: "Semantic PR"
-    runs-on: ubuntu-latest
+    runs-on: ${{ inputs.runs-on }}
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install prerequisites
+        run: sudo apt update && sudo apt install -y --no-install-recommends git unzip curl make
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)